### PR TITLE
Fix UNO 444 - Broken layers in DEMO when background tasks menu opened 

### PR DESCRIPTION
**Pull request by habemuscode**
**Needs merge and publish this PR's first**

https://github.com/oat-sa/ckeditor-dev/pull/20

**Related to**

https://oat-sa.atlassian.net/browse/UNO-444

**Description**

There're some issues with layers when the browser window resolution set to the smaller size e.g. 900x700 and the item has a property type set to Text - Long - HTML editor: the list of background tasks partially covered by edit item properties form. Reproduced only in `UDIR` demo environment.

**Preconditions**

Class property type is set to Text - Long - HTML editor.

**Steps to reproduce**

- Navigate to the Items tab
- Narrow the browser window 
- Open the Background tasks menu

**Actual result**

The opened Background tasks menu is partially covered by 'Kommentar' field in edit item properties filed.

**Expected result**

The opened Background tasks menu covers the edit item properties form.

imagen

### CODE REVIEW

**Review by Alroniks — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is `develop`

**Review by jsconan — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: lib/ckeditor/package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "ckeditor4-dev",
-	"version": "4.14.2",
+	"version": "4.14.3",
```
[the remark this answers is not in the document]
Comment (habemuscode) [reply lib/ckeditor/package-lock.json:3]: Yes, its strange but is generated when you run command

image

[the remark this answers is not in the document]
Comment (habemuscode) [reply lib/ckeditor/package-lock.json:3]: image

Looks like is used and then package.json only is moved to other place

[the remark this answers is not in the document]
Comment (jsconan) [reply lib/ckeditor/package-lock.json:3]: ok, I see.
anyway, not a big issue.

**Review by jsconan — APPROVED** [review]

**Review by andrey-ponamarev — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

**Pull request merged**